src/Compilers/VisualBasic/Portable/Symbols/SynthesizedSymbols/GeneratedNameConstants.vb

```diff
@@ -0,0 +1,57 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
+    Friend NotInheritable Class GeneratedNameConstants
+        Public Const DotReplacementInTypeNames As Char = "-"c
```

Comment:
>Public [](http://example.com/codeflow?start=8&length=6)

Consider using `Friend` for all of these constants, to minimize the differences with the previous iteration of `GeneratedNames.vb`, and also so the accessibility is clear from the fields directly.